lib/vsc/utils/generaloption.py

```diff
@@ -43,7 +43,7 @@
 from optparse import OptionParser, OptionGroup, Option, Values, HelpFormatter
 from optparse import BadOptionError, SUPPRESS_USAGE, NO_DEFAULT, OptionValueError
 from optparse import SUPPRESS_HELP as nohelp  # supported in optparse of python v2.4
-from optparse import _ as _gettext  # this is gettext normally
+from optparse import gettext as _gettext  # this is gettext.gettext normally
```

Comment:
Why not pull this from gettext.gettext and alias it as `_` as [suggested in the docs](https://docs.python.org/2/library/gettext.html#gettext.gettext).